npm install crash.

Crash when click on npm>install on windows :

```
events.js:292
      throw er; // Unhandled 'error' event
      ^

Error: spawn npm ENOENT
    at Process.ChildProcess._handle.onexit (internal/child_process.js:269:19)
    at onErrorNT (internal/child_process.js:465:16)
    at processTicksAndRejections (internal/process/task_queues.js:80:21)
Emitted 'error' event on ChildProcess instance at:
    at Process.ChildProcess._handle.onexit (internal/child_process.js:275:12)
    at onErrorNT (internal/child_process.js:465:16)
    at processTicksAndRejections (internal/process/task_queues.js:80:21) {
  errno: -4058,
  code: 'ENOENT',
  syscall: 'spawn npm',
  path: 'npm',
  spawnargs: [ 'i' ]
}
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! sailsense-dev@1.0.0 start:sailfi: `stack-monitor ./conf/sailfi-stack.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the sailsense-dev@1.0.0 start:sailfi script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\Bertrand\AppData\Roaming\npm-cache\_logs\2021-05-17T08_15_15_136Z-debug.log
```

--- a/server/models/Npm.js
+++ b/server/models/Npm.js
@@ -82,7 +82,7 @@ Npm.prototype.run = async function (command) {
     const args = defaultCommand[command]
       ? defaultCommand[command]
       : ['run', ...cmd]
-    return spawn('npm', args, { cwd: path })
+    return spawn('npm', args, { cwd: path, env: Object.assign({}, process.env), shell: true })
   }
 }
 

